apps/user-office-frontend-e2e/cypress/integration/proposal_administration.ts

```diff
@@ -286,11 +286,13 @@ context('Proposal administration tests', () => {
       }
       cy.contains('Proposals').click();
 
+      const token = window.localStorage.getItem('token');
+
       cy.request({
         url: '/download/pdf/proposal/1',
         method: 'GET',
         headers: {
-          authorization: `Bearer ${Cypress.env('SVC_ACC_TOKEN')}`,
+          authorization: `Bearer ${token}`,
```

Comment:
Should we also clean up the variable from `cypress.json` ?

Reply:
I don't think so because we are using the `SVC_ACC_TOKEN` in the `getE2EApi` which is used for all api calls that we execute while preparing the tests after database reset. At that point we can't use any other token because we are not logged in and this is the only way to execute the needed api calls.